sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/blob/perf/DownloadBlobTest.java

```diff
@@ -39,18 +42,31 @@ public void run() {
 
     static class NullOutputStream extends OutputStream {
```

Comment:
I think it would be better to make `NullOutputStream` a singleton, so all consumers can use the same instance instead of creating a new one.